src/sentry/models/groupsearchviewlastvisited.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.db.models import UniqueConstraint
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, region_silo_model
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_model
+class GroupSearchViewLastVisited(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Organization
+
+    user_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE")
+    organization = FlexibleForeignKey("sentry.Organization")
+    group_search_view = FlexibleForeignKey("sentry.GroupSearchView")
+
+    last_visited = models.DateTimeField(null=False, default=timezone.now)
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_groupsearchviewlastvisited"
+        constraints = [
+            UniqueConstraint(
+                fields=["user_id", "organization_id", "group_search_view_id"],
```

Comment:
jfyi: Since these are fks, you can refer to the column name vs the id, so `organization` instead of `organization_id`. 

You don't need to change it, both are fine.